internal/backend/oauth/oauth.go

```diff
@@ -54,9 +54,9 @@ func New(l *zap.Logger) sdkservices.OAuth {
 		)
 	}
 
-	githubAppsURLPart := os.Getenv("GITHUB_APPS_URL_PART")
-	if githubAppsURLPart == "" {
-		githubAppsURLPart = "apps"
+	appsDir := "apps"
+	if os.Getenv("GITHUB_ENTERPRISE_URL") != "" {
+		appsDir = "github-apps"
```

Comment:
Maybe this should be env var as well? Sounds like something dynamic

Reply:
It was an env var - but it turns out that it isn't dynamic, it's either "apps" for regular GH, or "github-apps" not GitHub Enterprise.